tastypie/bundle.py

```diff
@@ -32,4 +33,7 @@ def __init__(self,
         self.via_uri = via_uri
 
     def __repr__(self):
-        return "<Bundle for obj: '%s' and with data: '%s'>" % (self.obj, self.data)
+        repr_string = "<Bundle for obj: '%s' and with data: '%s'>"
```

Comment:
It's already unicode, because we're importing `unicode_literals`.

If we try to perform substitution first, we can get `UnicodeDecodeError`, because self.data or self.obj can contain non-ascii `str` (Django's models `repr` return `str` in Py2, converted from `__unicode__`-method). Maybe I should explicit call `repr` on `self.obj` and `self.data`?